libsolidity/codegen/ABIFunctions.cpp

```diff
@@ -1153,14 +1152,14 @@ string ABIFunctions::abiDecodingFunctionArray(ArrayType const& _type, bool _from
 		{
 			templ("staticBoundsCheck", "");
 			templ("retrieveElementPos", "add(offset, " + load + "(src))");
-			templ("baseEncodedSize", "0x20");
+			templ("stride", "0x20");
```

Comment:
Can this move out of the if now?

Reply:
Yes, I think so - even using the nicer ``_type.calldataStride()``, since the byte array case is excluded in this function anyways. I'll try and change it once you're done reviewing.